msg/json/en.json

```diff
@@ -396,5 +396,34 @@
 	"WORKSPACE_ARIA_LABEL": "Blockly Workspace",
 	"COLLAPSED_WARNINGS_WARNING": "Collapsed blocks contain warnings.",
 	"DIALOG_OK": "OK",
-	"DIALOG_CANCEL": "Cancel"
+	"DIALOG_CANCEL": "Cancel",
+	"DELETE_SHORTCUT": "Delete block (%1)",
+	"DELETE_KEY": "Del",
+	"EDIT_BLOCK_CONTENTS": "Edit Block contents (→︎)",
```

Comment:
Delete the (->) here and in `messages.js`. Populating those is the responsibility of the shortcut code, and will by dynamic.

Reply:
Same for all of these hints.